ts/IdentificationModal.tsx

```diff
@@ -140,8 +191,19 @@ class IdentificationModal extends React.PureComponent<Props, State> {
       );
     } else {
       // if the biometric is not available unlock the unlock code insertion
-      this.setState({ canInsertPin: true });
+      this.setState({ canInsertPinBiometry: true });
```

Comment:
could be `biometryAuthAvailable` a better name?
Now it's a little confusing what it means

Reply:
Yep, but need to invert all the boolean logic or call the variable `biometryAuthNotAvailable`.